src/SFA.DAS.ApplyService.Web.UnitTests/RoatpTaskListWorkflowServiceTests.cs

```diff
@@ -921,6 +921,50 @@ public void Whos_in_control_section_status_shows_as_complete_if_companies_house_
             status.Should().Be(TaskListSectionStatus.Completed);
         }
 
+        [Test]
+        public void Whos_in_control_section_status_shows_as_next_if_companies_house_verified_and_not_confirmed_and_who_is_in_control_is_false()
+        {
+            var organisationVerificationStatus = new OrganisationVerificationStatus
+            {
+                VerifiedCompaniesHouse = true,
+                VerifiedCharityCommission = false,
+                CompaniesHouseDataConfirmed = false,
+                CompaniesHouseManualEntry = true,
+                CharityCommissionDataConfirmed = false,
+                WhosInControlConfirmed = false
+            };
+
+            var applicationSequences = GenerateSectionsCompletedUpToWhosInControl();
+
+            var status = _service.SectionStatus(Guid.NewGuid(), RoatpWorkflowSequenceIds.YourOrganisation, RoatpWorkflowSectionIds.YourOrganisation.WhosInControl,
+                applicationSequences, organisationVerificationStatus);
+
+            status.Should().Be(TaskListSectionStatus.Next);
+        }
+
+
+        [Test]
+        public void Whos_in_control_section_status_shows_as_next_if_charity_commission_verified_and_not_confirmed_and_who_is_in_control_is_false()
+        {
+            var organisationVerificationStatus = new OrganisationVerificationStatus
+            {
+                VerifiedCompaniesHouse = true,
```

Comment:
should this be true as CompaniesHouseDataConfirmed is false so I think it will ask you to enter CH directors, not Trustees